### PR TITLE
Moves help menu to end, fixes Hotkeys Help

### PR DESCRIPTION
**Pull request by Tastyfish**
## About The Pull Request

- Moves the Help menu to the end of the menu bar, like every app that has ever had a Help menu.

image

- Makes Hotkeys Help actually do something, given the verb that menu item calls was removed 3 years ago.
  - It now show all of your hotkeys, as set in preferences, in a TGUI table. They're in Binding sort order, as opposed to the preferences window, which may be of use to people.

image

## Why It's Good For The Game

halp how do i standup

## Changelog

:cl:
add: Re-added the Hotkeys-Help verb, and linked the Hotkeys menu item to it.
qol: Move Help menu to the end of the menu bar.
/:cl:

### CODE REVIEW

**Comment by Tastyfish** [comment]
Waiting on review.

**Comment by Tastyfish** [comment]
Waiting on review.

File: interface/interface.dm [hunk 1]
```diff
@@ -109,3 +109,25 @@
...
+	for(var/key in key_bindings_by_key)
+		// Get the full names
+		var/list/binding_names = list()
+		for(var/kb_name in key_bindings_by_key[key])
+			var/datum/keybinding/binding = GLOB.keybindings_by_name[kb_name]
+			binding_names += binding.full_name
+
+		chat_message += "<b>[key]:</b> [jointext(binding_names, ", ")]\n"
+
+	to_chat(src, span_purple(chat_message))
```
Comment (LemonInTheDark) [inline interface/interface.dm:133]: I'm not sure how useful this really is, specially with the chat spam like you mentioned. What does this button do currently?

Comment (Tastyfish) [reply interface/interface.dm:133]: literally nothing

Comment (Tastyfish) [reply interface/interface.dm:133]: I believe the functionality was removed (but not the menu button) when we got configurable hotkeys.

Comment (LemonInTheDark) [reply interface/interface.dm:133]: You mentioned that the upside to using this over the keybindings menu in preferences was the difference in ordering right?

Comment (Tastyfish) [reply interface/interface.dm:133]: yes

**Comment by Tastyfish** [comment]
> Sorry I uh, I'd be happy with the behavior, it'd be nice to have it open a tgui window instead of just dumping to chat, but the idea's fine

I can do that in a basic table, sure.

**Comment by Tastyfish** [comment]
> Sorry I uh, I'd be happy with the behavior, it'd be nice to have it open a tgui window instead of just dumping to chat, but the idea's fine

@LemonInTheDark It's now in TGUI. See the main PR description for updated screenshot.

File: tgui/packages/tgui/styles/main.scss [hunk 2]
```diff
@@ -54,6 +54,7 @@
 @include meta.load-css('./interfaces/CrewManifest.scss');
 @include meta.load-css('./interfaces/ExperimentConfigure.scss');
 @include meta.load-css('./interfaces/HellishRunes.scss');
+@include meta.load-css('./interfaces/HotkeysHelp.scss');
```
Comment (Bizzonium) [inline tgui/packages/tgui/styles/main.scss:57]:
```suggestion
@include meta.load-css('./interfaces/HotKeysHelp.scss');
```
Without this it fails on linux.

Comment (Tastyfish) [reply tgui/packages/tgui/styles/main.scss:57]: whoops.

**Pull request merged**